File: halas/Dun_McDowell.pl
sub EVENT_ITEM { 
	if (plugin::check_handin(\%itemcount, 18762 => 1)) {
		quest::say("Ah.. ye wish to be a member o' the White Rose, then. eh? Well, let's train ye fer a bit. and see if ye've got what it takes.");
		quest::ding();
		quest::summonitem(13513); # torn white tunic
		quest::faction(275,5); # Rogues of the White Rose
		quest::exp(100);
	} 
	#do all other handins first with plugin, then let it do disciplines
	plugin::try_tome_handins(\%itemcount, $class, 'Rogue');
	plugin::return_items(\%itemcount);
}


#END of FILE Zone:halas  ID:29069 -- Dun_McDowell 

File: halas/Dun_McDowell.lua
function event_trade(e)
	local item_lib = require("items");
	if(item_lib.check_turn_in(e.trade, {item1 = 18762})) then
		e.self:Say("Ah.. ye wish to be a member o' the White Rose, then. eh? Well, let's train ye fer a bit. and see if ye've got what it takes.");
		e.other:Ding();
		e.other:SummonItem(13513);	-- torn white tunic
		e.other:Faction(275,100); 	-- Rogues of the White Rose
		e.other:AddEXP(100);
	end
	item_lib.return_items(e.self, e.other, e.trade)
end

-------------------------------------------------------------------------------------------------
-- Converted to .lua using MATLAB converter written by Stryd and manual edits by Speedz
-- Find/replace data for .pl --> .lua conversions provided by Speedz, Stryd, Sorvani and Robregen
-------------------------------------------------------------------------------------------------
